### DISCUSSION TITLE
Automatic for connect with v2ray

### DISCUSSION THREAD

**Discussion by OFzatt** [2dust/v2rayNG#2229] [Q&A]
Hello guys,
Im new here and have a question, hope get a help.

I want v2ray auto connect if my wifi cant connect to any network, and auto disconnect if wifi connect to any network. 

I've tried with automated tasks like bixby routines, automate, macrodroid... but I can't seem to find anything useful. Any suggestions, Thanks

**Comment by 666liuzi** [comment]
Have you solved this problem? If you solved it, can you provide a solution?

**Comment by padavano** [comment]
I did this using Macrodroid - I found the macros in the template repository. Two macros:
1. Connect v2ray when disconnected from network "wi-fi_name."
2. Disconnect v2ray when connected to network "wi-fi_name."

If you can't find them, I've attached the file - [v2ray-macro.zip](https://github.com/user-attachments/files/22410295/v2ray-macro.zip)
Just unzip it and import the macros. Then you need to change the name of the Wi-Fi network.
